addons/common/functions/fnc_displayTextStructured.sqf

```diff
@@ -24,52 +24,42 @@ params [["_text", ""], ["_size", 1.5, [0]], ["_target", ACE_player, [objNull]],
 if (_target != ACE_player) exitWith {};
 
 if (typeName _text != "TEXT") then {
-    if (_text isEqualType []) then {
-        if (count _text > 0) then {
-            {
-                if (_x isEqualType "" && {isLocalized _x}) then {
-                    _text set [_foreachIndex, localize _x];
-                };
-            }foreach _text;
-            _text = format _text;
-        };
+    if (_text isEqualType [] && {!(_text isEqualTo [])}) then {
+        {
+            if (_x isEqualType "" && {isLocalized _x}) then {
+                _text set [_forEachIndex, localize _x];
```

Comment:
`_text set [_forEachIndex` sounds like this should be a `apply`
```
format (_text apply {if (_x isEqualType "" && {isLocalized _x}) then { localize _x } else {_x} });
```
Not sure if the `()` are needed.
Is probably slower though. But'll look neater and this func doesn't seem like it's hot path anyway.